### PR TITLE
SET-718 Fix Zeus molecule issue

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/SET-718

molecule issue discussion: https://github.com/ansible/molecule/issues/4140

### CODE REVIEW

**Review by spyrkob — DISMISSED** [review]
Thanks, LGTM!

**Comment by spyrkob** [comment]
sigh that's not the PR I meant to approve... @rpelisse could you review this before you merge it?

File: molecule/account/molecule.yml [hunk 1]
```diff
@@ -9,16 +9,16 @@ driver:
...
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-    capabilities:
-      - SYS_ADMIN
-    command: "/usr/sbin/init"
+    image: registry.access.redhat.com/ubi8/ubi-init:latest
     pre_build_image: true
+    privileged: true
+    command: "/usr/sbin/init"
+    port_bindings:
```
Comment (rpelisse) [inline molecule/account/molecule.yml:16]: Why do we need port-binding? I think we can remove those line, don't you think?

**Review by rpelisse — CHANGES_REQUESTED** [review]

File: molecule/account/molecule.yml (hunk 1 above)
Comment (RanabirChakraborty) [reply molecule/account/molecule.yml:16]: You are right we are not testing any systemd-based service, we can remove it.

Comment (RanabirChakraborty) [reply molecule/account/molecule.yml]: @rpelisse modified it accordingly, Please review it.

**Review by rpelisse — APPROVED** [review]

**Pull request merged**